.eslintrc.js

```diff
@@ -9,8 +9,8 @@ module.exports = {
     '@vue/standard'
   ],
   rules: {
-    // 'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    // 'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
```

Comment:
Personally I would prefer to raise a 'no-debugger' error for production environments.

And as a future proposal - probably as part of a seperate PR: 
Maybe we could do something about the 'no-console' setting. E.g. we could pool all wegue logging operations into a logger module and then use the `eslint-disable` directive there. So warnings could be generally enabled.

Reply:
No problem to revert the change, simply, that's not something that I personally changed, this is what is present in `Vue-CLI V4` and `Vue-CLI V5` templates.

Your future proposal is intersting, it would be nice to enable this rule in the end for sure!

Reply:
Reverted to raise error if `debugger` is used while building for production